podcasts/Notifications/NotificationsCoordinator+AnalyticsAdapter.swift

```diff
@@ -58,6 +58,8 @@ extension NotificationType {
             possibleConditions = [.discoverListShowAllTapped]
         case .recommendationsTrending:
             possibleConditions = [.discoverListShowAllTapped]
+        case .upsell:
+            possibleConditions  = [.purchaseSuccessful]
```

Comment:
⛏️ Single extra space here.

```suggestion
            possibleConditions = [.purchaseSuccessful]
```